### ISSUE TITLE
[Feature] Tree table view.

### ISSUE THREAD

**Issue by RokeJulianLockhart** [TechbeeAT/jtxBoard#1365]
#### **Is your feature request related to a problem? Please describe.**

I want to be able to view multiple attributes of multiple tasks simultaneously.

#### **Describe the solution you'd like**

Implement a tree table view. It would need to be a table to demonstrate the attributes how I want, and would need to be a tree table to demonstrate hierarchy visually (like a file manager).

#### **Describe alternatives you've considered**

The current list view is the sole alternative.

#### **Additional context**

> https://github.com/tasks/tasks/issues/2854#issue-2262067601

#### **Device and version**

<table><tr>
	<th>Device</th>
	<td><code>cpe:/h:fairphone:fairphone_5</code></td>
</tr><tr>
	<th>Android version</th>
	<td>
		<table><tr>
			<th>Version</th>
			<td>13</td>
		</tr><tr>
			<th>Build</th>
			<td><code>FP5.TT3L.A.106.20230918</code></td>
		</tr><tr>
			<th>Downloaded from</th>
			<td>https://fairphone-android-builds.ams3.digitaloceanspaces.com/FP5/A13/FP5-TT4D-factory.zip</td>
		</tr></table>
	</td>
</tr><tr>
	<th>jtx Board version</th>
	<td><a href="https://f-droid.org/repo/at.techbee.jtx_207040005.apk">https://f-droid.org/repo/at.techbee.jtx_207040005.apk</a> (<a href="https://github.com/TechbeeAT/jtxBoard/releases/download/v2.7.5/jtxBoard-v2.7.5.apk">https://github.com/TechbeeAT/jtxBoard/releases/download/v2.7.5/jtxBoard-v2.7.5.apk</a>)</td>
</tr><tr>
	<th>Downloaded from</th>
	<td><a href="https://f-droid.org/packages/at.techbee.jtx/">https://f-droid.org/packages/at.techbee.jtx/</a></td>
</tr></table>

**Comment by patrickunterwegs** [comment]
Hello @RokeJulianLockhart , do you mean here a tree to display more then one level of subtasks?

**Comment by beedell-roke** [comment]
@patrickunterwegs, indeed. All subtasks should be visible simultaneously. NextCloud tasks  allows that, so precedent exists.
